docs/self-managed/zeebe-gateway-deployment/the-zeebe-gateway.md

```diff
@@ -1,94 +1,94 @@
 ---
 id: index
-title: "The Zeebe Gateway"
+title: "Zeebe gateway"
 sidebar_label: "Overview"
+description: "The Zeebe Gateway is a component of the Zeebe cluster."
 ---
 
-# Zeebe Gateway
+## What is the Zeebe gateway?
```

Comment:
Ah, I had something in mind about whether we need to start with `##`, but was not sure and haven't found it in the guides.

Reply:
Yes 👍 Always start with `##`.